x-pack/plugins/actions/server/lib/mustache_renderer.test.ts

```diff
@@ -340,4 +340,46 @@ describe('mustache_renderer', () => {
     const expected = '1 - {"c":2,"d":[3,4]} -- 5,{"f":6,"g":7}';
     expect(renderMustacheString('{{a}} - {{b}} -- {{e}}', deepVariables, 'none')).toEqual(expected);
   });
+
+  describe('converting dot variables', () => {
+    const dotVariables = {
+      context: [
+        {
+          'test.field': 1,
+        },
+        {
+          level1: {
+            'test.field': 2,
```

Comment:
Can we test a field with 2 dots and 3 dots?  I think change this one to have two, the one below to have 3.

Reply:
Resolved in this commit 00406ed0c744dff5775a571da58b0ba2d25295a6